zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -804,7 +804,7 @@ private static void sendMessage (Collection collection, List<String> uris, Strin
                 .log("traceId before sending event");
 
         try {
-            kafkaServiceSupplier.getService().produceContentPublished(collection.getId(), uris, dataType, traceId);
+            kafkaServiceSupplier.getService().produceContentPublished(collection.getId(), uris, dataType, "", "ONS", traceId);
```

Comment:
Should we be creating `public static final String SEARCHINDEX = "ONS";`

And then doing the following:

```suggestion
            kafkaServiceSupplier.getService().produceContentPublished(collection.getId(), uris, dataType, "", SEARCHINDEX, traceId);
```